app/services/validation_beta_service.rb

```diff
@@ -24,7 +24,7 @@ def invite_schools(array_of_urns:, participant_research:, coordinator_research:)
 
   def tell_induction_coordinators_to_check_ect_and_mentor_information
     InductionCoordinatorProfile.find_each do |ic|
-      if ic.schools.any? { |school| chosen_programme_and_not_in_beta(school) }
+      if ic.schools.any? { |school| chosen_programme_and_not_in_beta(school) && !opted_out_of_updates(school) }
```

Comment:
We have a scope on schools, `not_opted_out`.

That means we can do 
```
if ic.schools.not_opted_out.any? { |school| chosen_programme_and_not_in_beta(school) }
```
without the extra cumbersome check.
```